### PR TITLE
Add ``coalesce_brackets``

### PR DESCRIPTION
**Pull request by Asnelchristian**

### CODE REVIEW

**Comment by Adrianzatreanu** [comment]
ack 4c04de2171059caa9bbc105d75443d6c47b9085e

File: requirements.txt [hunk 1]
```diff
@@ -10,7 +10,7 @@ autoflake==0.6.*
 restructuredtext_lint==0.14.*
 proselint==0.3.*
 cpplint==1.*
-yapf==0.10.*
+yapf==0.11.*
```
Comment (Adrianzatreanu) [inline requirements.txt]: why?

Comment (Asnelchristian) [inline requirements.txt]: because this option is not supported by yapf~0.10.*

Comment (Adrianzatreanu) [inline requirements.txt]: oh alright!

**Comment by Adrianzatreanu** [comment]
ack a6597e8

**Comment by Adrianzatreanu** [comment]
@rultor merge

**Pull request merged**